sos/report/plugins/sedutil.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin, PluginOpt
+
+
+class SEDUtility(Plugin, IndependentPlugin):
+    """
+    Collects information about SED drives installed on host system.
+    This plugin will capture data using sedutil utility
+    """
+
+    short_desc = 'Self Encrypting Drives'
+    plugin_name = 'sedutil'
+    profiles = ('security', 'system', 'storage', 'hardware')
+    packages = ('sedutil',)
+
+    option_list = [
+        PluginOpt('debug', default=False, desc='capture debug data'),
+    ]
+
+    def setup(self):
+        sed_list = []
+        result = self.collect_cmd_output('sedutil-cli --scan')
+
+        if self.get_option("debug"):
+            if 0 == result['status']:
+                # iterate over all the devices returned and
+                # create a list of SED drives.
+                for line in result['output'].splitlines():
+                    line = line.strip()
+                    if line.startswith("/dev/"):
+                        line = " ".join(line.split())
+                        tcg_opal_dev = line.split(" ")[1]
+                        #  check if device is SED or not 
```

Comment:
`flake8 sos` :    `sos/report/plugins/sedutil.py:40:57: W291 trailing whitespace` (https://github.com/sosreport/sos/pull/3460/checks?check_run_id=20085390405)

Reply:
Done

Reply:
Oh yes my bad. Missed to change that.